RedHatInsights/insights-remediations-frontend
# fix: Fix notification link when patching a playbook

# Description

Associated Jira ticket: [# (issue)](https://issues.redhat.com/browse/RHINENG-4941)

Fix notification link when patching a playbook, previously it only sent user to Remediation app, now it directs him to detail page of the affected playbook.

# How to test the PR
1. Go to Vulnerability CVE detail page
2. Select a system with a checkbox
3. Click Remediate
4. Add system to some existing playbook
5. The displayed notification link should send you to detail page of that playbook

# Before the change
Only link in notification when new playbook was created worked.

# After the change
Links in both create and edit playbook notifications work.
